server.js

```diff
@@ -15,6 +17,99 @@ app.get("/", function (request, response) {
 
 // TODO add your routes and helper functions here
 
-const listener = app.listen(process.env.PORT, function () {
+// *******Get all bookings*****
+
+app.get("/bookings", (req, res) => {
+  res.json(bookings);
+});
+
+// *******Search booking with time span******
+
+app.get("/bookings/search", (req, res) => {
+  let date = req.query.date;
+  let matchedBookings = bookings.filter(
+    (book) => book.checkInDate <= date && book.checkOutDate >= date
+  );
+  if (matchedBookings.length > 0) {
+    res.json(matchedBookings);
+  } else res.send("No bookings found matching the chosen timespan");
```

Comment:
so this is great, however, omitting {} is ok
but try to be consistent 
so something like
 if ()
   do x
else 
  do y